### ISSUE TITLE
TestKillOneMasterNode, Elasticsearch_nodes_topology_should_be_the_expected_one

### ISSUE THREAD

**Issue by artemnikitin** [elastic/cloud-on-k8s#1090]
```go
17:17:08      --- FAIL: TestKillOneMasterNode/Elasticsearch_nodes_topology_should_be_the_expected_one#01 (0.02s)
17:17:08          require.go:157: 
17:17:08              	Error Trace:	checks_es.go:106
17:17:08              	Error:      	Not equal: 
17:17:08              	            	expected: 4
17:17:08              	            	actual  : 2
17:17:08              	Test:       	TestKillOneMasterNode/Elasticsearch_nodes_topology_should_be_the_expected_one#01
17:17:08  FAIL
```

**Comment by barkbay** [comment]
Should be fixed by https://github.com/elastic/cloud-on-k8s/pull/1083/commits/5604f9352677c070a07d73cb33e1b2aae923e8f8

**Comment by artemnikitin** [comment]
Yeah, looks like no such issue on very recent builds. Let's wait for 2-3 day to be sure and then close this issue.

**Comment by barkbay** [comment]
This one is also fixed, we are now using a retry loop to wait for the right topology.
fixed in #1078
